docs/ReleaseNotes.md

```diff
@@ -0,0 +1,77 @@
+# Antiplagiarism Release Notes
+This document provides information about fixes, enhancements, and key features that are available in Antiplagiarism.
```

Comment:
@k-s-popova please review release notes. Feel free to comit directly into the branch

Reply:
done